### PR TITLE
Add docs to bitrise.rb for help in setup with stash and bitrise

### PR DESCRIPTION
**Pull request by ihgmobile-devops**
Added a few lines of notes to help others setting up Danger with Bitrise and Bitbucket Server that might be using the Bitrise Close step a little out of the norm, like for firewall/vpn access from Bitrise to Bitbucket server.

### CODE REVIEW

**Comment by orta** [comment]
I'm not sure about the "see initialize below for the pattern" - because people won't be seeing the source code. It would show up at the bottom of this page: https://danger.systems/guides/getting_started.html

**Comment by orta** [comment]
Yeah, I think so

File: lib/danger/ci_source/bitrise.rb [hunk 1]
```diff
@@ -19,6 +19,14 @@ module Danger
   #
   # Add the `DANGER_GITHUB_API_TOKEN` to your workflow's [Secret App Env Vars](https://blog.bitrise.io/anyone-even-prs-can-have-secrets).
   #
+  # ### bitbucket server and bitrsie
+  #
+  # Danger will read the environemnt variable GIT_REPOSITORY_URL to construct the Bitbucket Server API URL 
+  # finding the project and repo slug in the GIT_REPOSITORY_URL variable. This GIT_REPOSITORY_URL variable 
+  # comes from the App Settings tab for your Bitrsie App. If you are manually setting a repo URL in the 
+  # Git Clone Repo step, you may need to set adjust this propery in the settings tab, maybe even fake it.
+  # see initialize below for the pattern
```
Comment (orta) [inline lib/danger/ci_source/bitrise.rb]:
```suggestion
  # The pattern used is `(%r{([\/:])(([^\/]+\/){1,2}[^\/]+?)(\.git$|$)}`.
```

Comment (orta) [reply lib/danger/ci_source/bitrise.rb]: I can't edit this repo, but if you accept this I can merge 👍

**Comment by danger-public** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Tests were not updated</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by orta** [comment]
Thanks!

**Pull request merged**